controllers/deviceController.js

```diff
@@ -0,0 +1,100 @@
+var express = require('express');
+var deviceController = express.Router();
+
+let data = {
```

Comment:
This is a some good work, but a lot of it is gonna be replaced by some SQL Tool for node functions that don't need to be written out.